parser.go

```diff
@@ -819,6 +819,19 @@ func (v *Value) GetUint64(keys ...string) uint64 {
 	return fastfloat.ParseUint64BestEffort(v.s)
 }
 
+// GetString returns the string value by the given keys path.
+//
+// Array indexes may be represented as decimal numbers in keys.
+//
+// "" is returned for non-existing keys path or for invalid value type.
+func (v *Value) GetString(keys ...string) string {
```

Comment:
I didn't add an equivalent `String` function because that would collide with the existing one that returns a string representation of the value.